src/main/scala/firrtl/annotations/PresetAnnotations.scala

```diff
@@ -0,0 +1,33 @@
+// See LICENSE for license details.
+
+package firrtl
+package annotations
+
+/**
+  * Transform all registers connected to the targeted AsyncReset tree into bitstream preset registers
+  * Impacts all registers connected to any child (cross module) of the target AsyncReset
+  *
+  * @param target ReferenceTarget to an AsyncReset
+  */
+case class PresetAnnotation(target: ReferenceTarget)
+    extends SingleTargetAnnotation[ReferenceTarget] with firrtl.transforms.DontTouchAllTargets {
+  override def duplicate(n: ReferenceTarget) = this.copy(target = n)
+}
+
+
+/**
+  * Transform the targeted asynchronously-reset Reg into a bitstream preset Reg
+  * Used internally to annotate all registers associated to an AsyncReset tree
```

Comment:
If this is only used internally then should it be `private[firrtl]`? Like this shouldn't ever be leaked from the `VerilogEmitter` so it shouldn't need to be public right?

Reply:
fixed in rebase